README.md

```
@@ -244,12 +244,32 @@ The framework dispatches a custom event when all modules are loaded:
 
 ### Debug Information
 
+**Enable debug logging** by adding `?debug` to your URL:
+
+```
+Visit: https://yoursite.com/?debug
+```
+
+When debug mode is active, you'll see structured logs in the console:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)